contrib/test/ci/cri-o.spec

```diff
@@ -0,0 +1,189 @@
+# This spec is meant for CI testing only.
+# Any changes here will NOT automagically land in your distro's rpm after
+# an update.
+
+%global with_debug 1
+
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+
+%if ! 0%{?centos} && 0%{?rhel}
+# Golang minor version
+%global gominver 10
+%define gobuild(o:) scl enable go-toolset-1.%{gominver} -- go build -buildmode pie -compiler gc -tags="rpm_crashtraceback no_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v -x %{?**};
+%else
+%define gobuild(o:) go build -buildmode pie -compiler gc -tags="rpm_crashtraceback no_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v -x %{?**};
+%endif
+
+%global provider github
+%global provider_tld com
+%global project cri-o
+%global repo cri-o
+# https://github.com/kubernetes-sigs/cri-o
```

Comment:
nit, it's been a long time since we lived in kubernetes-sigs :)

Reply:
haha good catch :) fixed